lego-webapp/pages/achievements/+Page.tsx

```diff
@@ -102,6 +102,7 @@ const AchievementGroup = ({
                     gap="var(--spacing-sm)"
                     alignItems="center"
                     justifyContent="center"
+                    align-text="center"
```

Comment:
Is this supported? It should be camelCase

Reply:
I think you should fix this in the `TitleWithRarity` component instead of here. Also I think just adding `wrap` to the flexbox there solves it pretty good. Also a `justify-content='center'`